### ISSUE TITLE
Issue with master

### ISSUE THREAD

**Issue by arnodelorme** [sccn/roiconnect#32]
```
Undefined function or variable 'desired_nfreq'.

Error in data2cs_event (line 114)
        if desired_nfreq ~= 0

Error in data2spwctrgc (line 108)
    CS = data2cs_event(data(:, :)',ndat, ndat - (floor(ndat/2)), ndat , [], CSpara);

Error in roi_connect (line 121)
        conn_mult = data2spwctrgc(source_roi_data, EEG.pnts, g.morder, 0, g.naccu, [], tmpMethods2);

Error in pop_roi_connect (line 244)
    EEG = roi_connect(EEG, 'morder', g.morder, 'naccu', g.naccu, 'methods', g.methods,'freqresolution',g.freqresolution);

```

**Comment by arnodelorme** [comment]
Partially fixed in [6f45bc1](https://github.com/sccn/roiconnect/commit/6f45bc191db6b63d61f4a3f694470b7709bbf7d3)

**Comment by Hiyeri** [comment]
Should be fixed with #36
